### PR TITLE
Intersect the ordering of parents and job groups

### PR DESCRIPTION
**Pull request by coolo**
We want some groups at the top and the dead groups parent at the end

### CODE REVIEW

**Comment by okurz** [comment]
+1

**Comment by Martchus** [comment]
Looks good.

**Comment by Martchus** [comment]
I will adapt the editor.

File: lib/OpenQA/Utils.pm [hunk 1]
```diff
@@ -513,5 +514,26 @@ sub human_readable_size {
     return $p . _round_a_bit($size) . "GiB";
 }
 
+# query group parents and job groups and let the database sort it for us - and merge it afterwards
+sub job_groups_and_parents {
```
Comment (coolo) [inline lib/OpenQA/Utils.pm]: actually I would love to move this to a ResultSet class, but I can't decided which one :)

**Pull request merged**